internal/cli/atlas/deployments/options/deployment_opts.go

```diff
@@ -98,10 +102,16 @@ type Deployment struct {
 	StateName      string
 }
 
-func (opts *DeploymentOpts) InitStore(podmanClient podman.Client) func() error {
+func (opts *DeploymentOpts) InitStore(ctx context.Context, podmanClient podman.Client) func() error {
 	return func() error {
+		var err error
 		opts.PodmanClient = podmanClient
-		return nil
+		opts.Config = config.Default()
+		opts.CredStore = config.Default()
+		if opts.AtlasClusterListStore, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx)); err != nil {
```

Comment:
question, should this be done only if `opts.IsCliAuthenticated()`?

Reply:
i think init won't fail if user is not authenticated so we could defer the usage to later?

Reply:
this might fail, make sure to test without setting your default profile